js/src/elements/form-behavior.ts

```diff
@@ -364,6 +366,16 @@ export class FormBehaviorElement extends HTMLElement {
 		return null;
 	}
 
+	validateGroupRequired(input: HTMLValueElement, label: string): string | null {
+		const group = getField(input);
+		const inputs = Array.from(group.querySelectorAll<HTMLInputElement>('input'));
+		const selected = inputs.some(input => input.checked);
+		if (!selected) {
+			return `${this.locStrings.inputGroupRequired.replace('{inputLabel}', label)}`;
+		}
+		return null;
```

Comment:
Do we need this `return null` if we're already at the end of the function?

Reply:
I believe the null allows for fall-through checks on the validators iteration.  Previous implementation pattern present with other validators too.